src/ComponentInstaller.php

```diff
@@ -295,7 +269,12 @@ private function findApplicationModules()
                     continue;
                 }
 
-                $modules[] = $file->getBasename();
+                $module = $file->getBaseName();
+                if (empty($module)) {
+                    continue;
+                }
```

Comment:
Yikes, this can happen?!?! Good check!

Reply:
I guess thats not the case but I prefer having real checks over psalm-only annotations.
The return value is `non-empty-string` so I just ensure that psalm believes me 👍🏼